ash-window/src/lib.rs

```diff
@@ -170,13 +174,7 @@ pub fn enumerate_required_extensions(
 ) -> VkResult<&'static [*const c_char]> {
     let extensions = match window_handle.raw_window_handle() {
         #[cfg(target_os = "windows")]
-        RawWindowHandle::Windows(_) => {
-            const WINDOWS_EXTS: [*const c_char; 2] = [
-                khr::Surface::name().as_ptr(),
-                khr::Win32Surface::name().as_ptr(),
-            ];
-            &WINDOWS_EXTS
-        }
+        RawWindowHandle::Windows(_) => &WINDOWS_SURFACE_EXTENSIONS,
```

Comment:
Is the `&` here and below needed?

Reply:
I don't think so, I'll fix that.

Reply:
Don't we have clippy calling [`needless_borrow`](https://rust-lang.github.io/rust-clippy/master/#needless_borrow) on this?

If not, that's bad.